src/resources/js/tickets.js

```diff
@@ -3,17 +3,17 @@ var ticketHeaderImage = window.ticketHeaderImage || {};
 (function( window, $, undefined ) {
 	'use strict';
 
-	var $event_pickers               = $( document.getElementById( 'tribe-event-datepickers' ) );
-	var $tribe_tickets               = $( document.getElementById( 'tribetickets' ) );
-	var $tickets_container           = $( document.getElementById( 'event_tickets' ) );
-	var $enable_global_stock         = $( document.getElementById( 'tribe-tickets-enable-global-stock' ) );
-	var $global_stock_level          = $( document.getElementById( 'tribe-tickets-global-stock-level' ) );
-	var global_stock_setting_changed = false;
-	var $body                        = $( 'html, body' );
-	var startofweek                  = 0;
-	var $base_panel                  = $( document.getElementById( 'panel_base' ) );
-	var $edit_panel                  = $( document.getElementById( 'panel_edit' ) );
-	var $settings_panel              = $( document.getElementById( 'panel_settings' ) );
+	var $event_pickers               = $( document.getElementById( 'tribe-event-datepickers' ) ),
+		$tribe_tickets               = $( document.getElementById( 'tribetickets' ) ),
```

Comment:
As noted in [our engineering standards](http://moderntribe.github.io/products-engineering/js/), we should probably stick to one statement per line for these declarations.